server/remote_cache/hit_tracker/hit_tracker.go

```diff
@@ -132,7 +126,7 @@ func (h *HitTracker) TrackMiss(d *repb.Digest) error {
 		metrics.CacheTypeLabel:      h.cacheTypeLabel(),
 		metrics.CacheEventTypeLabel: missLabel,
 	}).Inc()
-	_, err := h.c.IncrementCount(h.ctx, h.counterName(Miss), 1)
+	err := h.c.IncrementCount(h.ctx, h.counterName(Miss), 1)
 	return err
```

Comment:
you can just return h.c.IncrementCount(h.ctx, h.counterName(Miss), 1) directly, here and below

Reply:
Done